src/Analyzer.BicepProcessor.UnitTests/BicepProcessorUnitTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Azure.Templates.Analyzer.BicepProcessor;
+using Microsoft.Azure.Templates.Analyzer.Core;
+using Microsoft.Azure.Templates.Analyzer.Types;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Analyzer.BicepProcessor.UnitTests
+{
+    [TestClass]
+    public class BicepProcessorUnitTests
+    {
+        [TestMethod]
+        public void ValidateBicepModuleMetadata()
```

Comment:
```suggestion
        public void ValidateBicepModuleMetadataCounts()
```
If I understood the assert?

Reply:
The next line does the comparison of the actual modules so it validates more than counts. The combination of dict1.Count == dict2.Count && !(dict1.Except(dict2).Any()) is a shorthand for comparing two dictionaries.